packages/datastore/src/query.js

```diff
@@ -311,27 +333,16 @@ Query.prototype.offset = function(n) {
  * // If you anticipate many results, you can end a stream early to prevent
  * // unnecessary processing and API requests.
  * //-
- * query.run()
+ * query.stream()
  *   .on('data', function (entity) {
  *     this.end();
  *   });
- *
- * //-
- * // A keys-only query returns just the keys of the result entities instead of
- * // the entities themselves, at lower latency and cost.
- * //-
- * query.select('__key__');
- *
- * query.run(function(err, entities) {
- *   // entities[].key = Key object
- *   // entities[].data = Empty object
- * });
  */
-Query.prototype.run = function() {
+Query.prototype.stream = function() {
```

Comment:
I'm not sure about `query.run` and `query.stream`. I would think `query.runStream()` to match our convention.